composer.json

```diff
@@ -1,32 +1,41 @@
 {
-    "name": "yiisoft/_____",
+    "name": "yiisoft/auth",
     "type": "library",
-    "description": "_____",
-    "keywords": ["_____"],
+    "description": "Yii auth",
+    "keywords": ["auth", "middleware"],
     "homepage": "http://www.yiiframework.com/",
     "license": "BSD-3-Clause",
     "support": {
-        "issues": "https://github.com/yiisoft/_____/issues?state=open",
+        "issues": "https://github.com/yiisoft/auth/issues?state=open",
         "forum": "http://www.yiiframework.com/forum/",
         "wiki": "http://www.yiiframework.com/wiki/",
         "irc": "irc://irc.freenode.net/yii",
-        "source": "https://github.com/yiisoft/_____"
+        "source": "https://github.com/yiisoft/auth"
     },
+    "minimum-stability": "dev",
     "require": {
-        "php": "^7.2"
+        "php": "^7.2",
+        "psr/container": "^1.0",
```

Comment:
Is it possible to get rid of this dependency? It's not bad as such, just wondering.